driver/csiplugin/controllerserver.go

```diff
@@ -591,20 +592,28 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 		scaleVol.LocalFS = scaleVol.VolBackendFs
 	}
 
+	var remoteClusterID string
+	if volFsInfo.Type == filesystemTypeRemote {
```

Comment:
I think we can add the additional condition here something like 
if scaleVol.ClusterId == "" && volFsInfo.Type == filesystemTypeRemote

Reply:
Added.